esp/src/ganglia/nls/es/ganglia.js

```diff
@@ -0,0 +1,13 @@
+define(
+({
+    AddMetric: "",
```

Comment:
Its better to exclude any untranslated variables as they will fall back to use the English (otherwise they will display blank)